client/connect-account-page/strings.tsx

```diff
@@ -285,4 +285,21 @@ export default {
 			'woocommerce-payments'
 		),
 	},
+	nonSupportedCountry: createInterpolateElement(
+		__(
+			'<b>WooPayments is not currently available in your location</b>. To be eligible for WooPayments, your business address must be in one of the following <a>supported countries</a>.',
```

Comment:
We could keep the way that we extracted `WooPayments` outside of the strings like we did in the Branding rollout, what do you think? Example: https://github.com/Automattic/woocommerce-payments/blob/f0d1ae05973b9e26e0dae4e8652357b3db8cd470/client/onboarding/strings.tsx#L41